pymc4/distributions/continuous.py

```diff
@@ -974,6 +975,63 @@ def _init_distribution(conditions):
         return tfd.LogNormal(loc=loc, scale=scale)
 
 
+class Moyal(ContinuousDistribution):
+    r"""Continuous Moyal random variable.
+
+    The pdf of this distribution is
+
+    .. math::
+
+        f(x \mid \mu, \sigma) = 
+           \frac{1}{\sqrt{2\pi}\sigma}
+           \exp\left(-\frac{1}{2}\left[\frac{x-\mu}{\sigma}+\exp\left(-\frac{x-\mu}{\sigma}\right)\right]\right)
+
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import scipy.stats as st
+        plt.style.use('arviz-darkgrid')
+        x = np.linspace(-5, 5, 1000)
```

Comment:
nitpicking: Could you use `x = np.linspace(-5, 8, 1000)` or `x = np.linspace(-5, 10, 1000)`?

Reply:
done